SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -459,30 +481,107 @@ void USpatialNetDriver::CreateServerSpatialOSNetConnection()
 	GetWorld()->SpatialProcessServerTravelDelegate.BindStatic(SpatialProcessServerTravel);
 }
 
-void USpatialNetDriver::QueryGSMToLoadMap()
+bool USpatialNetDriver::CanSendPlayerSpawnRequests()
```

Comment:
This is only supposed to be called on clients right? If so, can we prefix `Client` to the function name?